### PR TITLE
Revert CodeMirror to 4.11.0

### PR DESCRIPTION
**Pull request by sindhus**

### CODE REVIEW

File: baseframe/Makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Makefile for tinymce
 all: tinymce codemirror
 
-tinymce3_zip = tinymce_3.5.11_jquery.zip
 tinymce4_zip = tinymce_4.3.0_jquery.zip
-codemirror_ver = 5.8.0
+tinymce4_zip = tinymce_4.1.7_jquery.zip
```
Comment (jace) [inline baseframe/Makefile]: This shouldn't be here.

Comment (sindhus) [inline baseframe/Makefile]: When I pulled and rebased from master with `git pull --rebase baseframe master`, `codemirror_ver` was right in between tinymce4 and tinymce5, so I moved the variable below which is why git shows it as a new addition.

Comment (jace) [inline baseframe/Makefile]: Look again. You've removed TinyMCE 3 and inserted a second copy for 4.

Comment (sindhus) [inline baseframe/Makefile]: Ok will fix.

**Pull request closed without merging**